jerry-core/parser/js/js-parser.c

```diff
@@ -2928,6 +2958,126 @@ parser_parse_script (const uint8_t *arg_list_p, /**< function argument list */
 #endif /* !JERRY_DISABLE_JS_PARSER */
 } /* parser_parse_script */
 
+#ifndef CONFIG_DISABLE_ES2015_MODULE_SYSTEM
+/**
+ * Parse EcamScript module source code
+ *
+ * Note:
+ *      if arg_list_p is not NULL, a function body is parsed
+ *      returned value must be freed with ecma_free_value
+ *
+ * @return true - if success
+ *         syntax error - otherwise
+ */
+ecma_value_t
+parser_parse_module_script (const uint8_t *arg_list_p, /**< function argument list */
```

Comment:
This function is almost the duplicate of `parser_parse_script`. Cannot these function can be merged somehow?